test/rubocop/cop/minitest/focus_test.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require_relative '../../../test_helper'
+
+class FocusTest < Minitest::Test
+  def test_registers_offense_when_using_direct_focus
+    assert_offense(<<~RUBY)
+      class FooTest < Minitest::Test
+        focus test 'foo' do
+        ^^^^^^^^^^^^^^^^ Remove `focus` from tests.
+        end
```

Comment:
The logic above can clarify the target:

```suggestion
        focus test 'foo' do
        ^^^^^ Remove `focus` from tests.
        end
```